### DISCUSSION TITLE
Protected notes are still visible after a long time

### DISCUSSION THREAD

**Discussion by alexios-y** [standardnotes/forum#2764] [Suggestions]
I am using the latest version of the mac app. Normally I will choose to remember me for 5 minutes when trying to view protected notes. However, the encryption is indeed reenabled after that period but i will still be able to see the protected note if I stayed on the note. It is not the ideal situation. Ideally, when I came back to the app, it should ask me to reenter my password rather than just display the protected one.

**Comment by atmoio** [comment]
Hmm yeah, good point, we'll take a look at this. It's a bit tricky though if you're editing the note and your 5 minute timer expires. We can either immediately lock it, or wait until you haven't edited the note for that same period of time.

**Comment by alexios-y** [comment]
> Hmm yeah, good point, we'll take a look at this. It's a bit tricky though if you're editing the note and your 5 minute timer expires. We can either immediately lock it, or wait until you haven't edited the note for that same period of time.

It indeed sounds a bit tricky. Maybe you could detect if a user is editing the note before locking it after timer expires. Nonetheless, I don’t feel the status quo is secure enough with this issue persists.
